parachains/runtimes/assets/asset-hub-kusama/src/xcm_config.rs

```diff
@@ -503,3 +531,102 @@ impl pallet_assets::BenchmarkHelper<MultiLocation> for XcmBenchmarkHelper {
 		MultiLocation { parents: 1, interior: X1(Parachain(id)) }
 	}
 }
+
+/// All configuration related to bridging
+pub mod bridging {
+	use super::*;
+	use assets_common::{matching, matching::*};
+	use sp_std::collections::btree_set::BTreeSet;
+	use xcm_builder::{NetworkExportTable, UnpaidRemoteExporter};
+
+	parameter_types! {
+		pub BridgeHubKusamaParaId: u32 = 1002;
+		pub BridgeHubKusama: MultiLocation = MultiLocation::new(1, X1(Parachain(BridgeHubKusamaParaId::get())));
+		pub BridgeHubKusamaWithBridgeHubPolkadotInstance: MultiLocation = MultiLocation::new(1, X2(Parachain(BridgeHubKusamaParaId::get()), PalletInstance(53)));
```

Comment:
Where is this type for? I might not understand what exactly is happening here, but BridgeHubKusama does not have a pallet instance 53, right? Or am I completely mistaken?

Reply:
yes, now BridgeHubKusama/Polkadot are just empty runtimes (in master),
but real bridging comes here: https://github.com/paritytech/cumulus/pull/2528

and here is **53 pallet**: https://github.com/paritytech/cumulus/pull/2528/files#diff-66b0425b5f194d88a1c1926558bccae96e56d6088b2123ed855b595a917997d3R536

this addition of PalletInstance was requirement of security audit